ios/RCTContacts/RCTContacts.m

```diff
@@ -386,14 +405,14 @@ -(NSString *) getFilePathForThumbnailImage:(NSString *)recordID
 
     [self updateRecord:contact withData:contactData];
 
-    CNContactViewController *controller = [CNContactViewController viewControllerForNewContact:contact];
-
-    controller.delegate = self;
+    CNContactViewController *contactViewController = [CNContactViewController viewControllerForNewContact:contact];
```

Comment:
Maybe just revert these lines to keep the PR shorter. No actual changes have been made except variable name changes.

Reply:
sure